docs/user-guide/sns-instruments/vanadium_processing.ipynb

```diff
@@ -0,0 +1,395 @@
+    "params = {\n",
+    "    NeXusDetectorName: \"powgen_detector\",\n",
+    "    # Input data\n",
+    "    Filename[VanadiumRun]: powgen.data.powgen_tutorial_vanadium_file(),\n",
+    "    # Edges for binning in d-spacing\n",
+    "    DspacingBins: sc.linspace(\"dspacing\", 0.0, 2.3434, 5001, unit=\"angstrom\"),\n",
+    "    # Mask in time-of-flight to crop to valid range\n",
+    "    TofMask: lambda x: (x < sc.scalar(0.0, unit=\"us\"))\n",
+    "    | (x > sc.scalar(16666.67, unit=\"us\")),\n",
+    "    TwoThetaMask: None,\n",
+    "    WavelengthMask: None,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f59bf8e0-72a4-4ee5-8217-4b1fa589cca4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "providers = [*powder.providers, *powgen.providers]\n",
+    "pipeline = sciline.Pipeline(providers, params=params)\n",
+    "pipeline = powder.with_pixel_mask_filenames(pipeline, [])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aa3cfbe6-7a7f-485b-9ea5-f3aad793f713",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline.visualize(FocussedDataDspacing[VanadiumRun], graph_attr={\"rankdir\": \"LR\"})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12ac6517-a73c-493b-b2ce-f7ffe13a2060",
+   "metadata": {},
+   "source": [
+    "Compute a single vanadium spectrum in d-spacing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "646a1117-3a71-4893-b2a2-13675cf749c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "peaked_data = pipeline.compute(FocussedDataDspacing[VanadiumRun])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8de3d6e0-b8ec-4a25-bcb0-b0d2b19b8a61",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "peaked_data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "04f7f83c-8608-458d-b4a9-2f9853c09cd7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "peaked_data.hist().plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "84f8b41c-e3f3-4282-83c6-e0e424bc5632",
+   "metadata": {},
+   "source": [
+    "## Removing coherent scattering peaks\n",
+    "\n",
+    "As the variable name `peaked_data` implies, the produced spectrum contains peaks from coherent scattering.\n",
+    "Even though the peaks are small for vanadium, we need to remove them to extract pure incoherent scattering.\n",
+    "We can approximate the coherent scattering contribution by fitting functions to the peaks and subtracting those fitted functions.\n",
+    "[scippneutron.peaks](https://scipp.github.io/scippneutron/generated/modules/scippneutron.peaks.html) contains general functionality for fitting and removing peaks.\n",
+    "Here, we use it through [ess.powder.external.powgen.peaks](../../generated/modules/ess.powder.external.powgen.peaks.rst) which provides useful defaults for vanadium peaks at POWGEN.\n",
```

Comment:
Once #68 will be merged, the path will be `ess.snspowder.powgen`. The files should also be moved around. Hopefully this won't be too annoying.